digits/dataset/tasks/create_db.py

```diff
@@ -191,19 +189,19 @@ def process_output(self, line):
             if not hasattr(self, 'distribution') or self.distribution is None:
                 self.distribution = {}
 
-            self.distribution[match.group(1)] = int(match.group(2))
-
-            data = self.distribution_data()
-            if data:
-                socketio.emit('task update',
-                              {
-                                  'task': self.html_id(),
-                                  'update': 'distribution',
-                                  'data': data,
-                              },
-                              namespace='/jobs',
-                              room=self.job_id,
-                              )
+            self.distribution[match.group(1)] = [int(match.group(2)), 0]
+            self.update_distribution_graph()
+            return True
+
+        # add errors to the distribution
+        match = re.match(r'\[(.+) (\d+)\] LoadImageError: (.+)', message)
+        if match:
+            self.distribution[match.group(2)][0] -= 1
+            self.distribution[match.group(2)][1] += 1
+            if self.entries_error is None:
```

Comment:
I tried to mimic the `self.entries_count`, but I can change to init to 0 if you think it is better